include/triton/Analysis/AxisInfo.h

```diff
@@ -205,6 +205,15 @@ class ModuleAxisInfoAnalysis : public CallGraph<AxisInfoMapT> {
   unsigned getContiguity(Value value);
   unsigned getAlignment(Value value);
 
+  // Overloads of the above methods but have separated elementBitWidth to
+  // calculate the contiguity. These are useful for computing axis info when
+  // lowering to hardware intrinsics that require a scalar/warp-uniform base ptr
+  // with separate per lane offsets, e.g. AMD buffer operations. For those cases
+  // we want to compute the contiguity on the offsets but use the pointee type
+  // element bit width instead of the offset element type bit width
```

Comment:
Nit: adding an example would make it even clear: ".. per lane offsets like AMD buffer operations. As a concrete example, instead of a single `tensor<128x64x!tt.ptr<f16>>` value, now we have two separate values: `!tt.ptr<f16>` for the base pointer and `tensor<128x64xi32>` for the offset. For such cases, we want to .."

Reply:
I agree with adding examples in the comment